cmake/Dependencies.cmake

```
@@ -1909,13 +1909,15 @@ if(USE_KINETO)
         ${CUDA_SOURCE_DIR}/extras/CUPTI/lib64
         ${CUDA_SOURCE_DIR}/lib
         ${CUDA_SOURCE_DIR}/lib64
+        $ENV{CUPTI_LIBRARY_DIR}
```

I'm not sure how CMake treats spaces in environment variables, but if it were a bash script following would prevent variable splitting (would you mind testing for it, and mention it in the test plan)
```suggestion
        "$ENV{CUPTI_LIBRARY_DIR}"
```